qvm/template-gui.jinja

```diff
@@ -45,7 +45,11 @@
         admin.vm.device.pci.Available       *   {{ vmname }}             dom0                             allow
         admin.vm.device.mic.Available       *   {{ vmname }}             dom0                             allow
         admin.vm.feature.Get        +internal   {{ vmname }}             dom0                             allow
-        admin.vm.feature.Get        +servicevm  {{ vmname }}             dom0                             allow
+        admin.vm.feature.Get       +servicevm  {{ vmname }}             dom0                             allow
+        admin.vm.feature.Get        +template   {{ vmname }}             @tag:guivm-{{ vmname }}          allow
+        # TODO: limit to templates related to @tag:guivm-{{ vmname }} only
+        admin.vm.feature.Get        +template   {{ vmname }}             @type:TemplateVM                 allow
```

Comment:
This one doesn't make sense, TemplateVM does not have a `template` property...

Reply:
Oh, and even more so - there is no `template` _feature_ at all, so the earlier line is wrong too...

Reply:
I guess something checks recursively for templates. IMO it's okay to keep this call denied, but **if** that causes issues, the code should be adjusted to deal with not having access to this non-existing property. My guess is it's [this loop](https://github.com/QubesOS/qubes-manager/blob/main/qubesmanager/settings.py#L440-L448), and it uses `hasattr` already so all should be fine.